apps/api/app/Domain/Bank/Actions/SummarizeBankAccount.php

```
@@ -0,0 +1,194 @@
+            ->findMany($pending->pluck('invoice_id'))
+            ->keyBy('id');
+
+        $ordered = $pending
+            ->each(static fn (BankMatch $match) => $match->setRelation('invoice', $invoices->get($match->invoice_id)))
+            ->sortBy([
+                static fn (BankMatch $a, BankMatch $b): int => $b->movement->booked_on->getTimestamp() <=> $a->movement->booked_on->getTimestamp(),
+                static fn (BankMatch $a, BankMatch $b): int => $b->id <=> $a->id,
+            ])
+            ->values();
+
+        return array_values(BankMatchData::collect($ordered, 'array'));
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\Domain\Bank\Data\BankMatchData' in method 'pendingMatches'. (undefined)

(StaticAccess)